business/istio_validations.go

```diff
@@ -157,42 +153,40 @@ func (in *IstioValidationsService) GetIstioObjectValidations(ctx context.Context
 	errChan := make(chan error, 1)
 
 	// Get all the Istio objects from a Namespace and all gateways from every namespace
-	wg.Add(9)
+	wg.Add(7)
 	go in.fetchNamespaces(ctx, &namespaces, errChan, &wg)
-	go in.fetchIstioConfigList(ctx, &istioConfigList, namespace, errChan, &wg)
-	go in.fetchExportedResources(ctx, &exportedResources, namespace, errChan, &wg)
+	go in.fetchIstioConfigList(ctx, &istioConfigList, &mtlsDetails, &rbacDetails, namespace, errChan, &wg)
 	go in.fetchServices(ctx, &services, namespace, errChan, &wg)
```

Comment:
This "var services models.ServiceList" is populated but it seems it is not used in the context of the "GetIstioObjectValidations", isn't it?